index.html

```diff
@@ -33,13 +47,22 @@
           <button type="button" class="btn btn-warning" id="first_capture" tabindex='-1'>First Version</button>
         </td>
     	</tr>
+      <tr class="spaceUnder" align="center">
+        <td>
+           <a href="https://www.facebook.com/sharer/sharer.php?u=" class="btn-social btn-sm btn-facebook"><i class="fa fa-facebook"></i></a>
+            <a href="https://twitter.com/home?status=" class="btn-social btn-sm btn-twitter"><i class="fa fa-twitter"></i></a>
+            <a href="https://plus.google.com/share?url=" class="btn-social btn-sm btn-google-plus"><i class="fa fa-google-plus"></i></a>
+         </td>
+    	</tr>
+  		
     </table>
     <div style="margin: 0px; padding: 0px; border: 0px solid rgb(6, 117, 211); font-size: 19px; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; line-height: 1; font-family: sans-serif; vertical-align: baseline; box-sizing: content-box; overflow: unset; height: 1px; position: relative; width: auto; display: flex; background-color: rgb(9, 150, 248); color: rgb(255, 255, 255); text-align: left; align-items: center; border-radius: 3px; justify-content: center; text-decoration: none;"></div>
   </body>
 </html>
-<script src="scripts/jquery.js"></script>
+<script src="scripts/jquery.min.js"></script>
```

Comment:
I completely agree to you on this for performance. I also prefer java script for performance 
but then we will have error in console if you check because bootstrap's javascript needs jquery.
![screenshot from 2017-03-29 11 43 40](https://cloud.githubusercontent.com/assets/11758140/24440999/108d99ce-1475-11e7-85a0-63c4218aa513.png)
 
i reported isse before regarding same [issue#13](https://github.com/internetarchive/wayback-machine-chrome/issues/13)
 can u please guide me on that ?